substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/SubstrateOptions.java

```diff
@@ -587,6 +587,11 @@ protected void onValueUpdate(EconomicMap<OptionKey<?>, Object> values, Boolean o
                 }
             }
         };
+
+        @APIOption(name = "install-exit-handlers", deprecated = "Enabled by default for executables. For shared libraries (when --shared is used), use the experimental option -H:+InstallJavaExitHandlersForSharedLibrary.")//
+        @Option(help = "Provide java.lang.Terminator exit handlers", deprecated = true, deprecationMessage = "Enabled by default for executables. For shared libraries (when --shared is used), use the experimental option -H:+InstallJavaExitHandlersForSharedLibrary.")//
+        protected static final HostedOptionKey<Boolean> InstallExitHandlers = new HostedOptionKey<>(false);
```

Comment:
Shouldn't this be true?

```suggestion
        protected static final HostedOptionKey<Boolean> InstallExitHandlers = new HostedOptionKey<>(true);
```